Makefile

```diff
@@ -306,12 +306,50 @@ manifests:  $(STAGE)-version-check $(STAGE)-flavors $(MANIFEST_DIR) $(BUILD_DIR)
 	"$(KUSTOMIZE)" build $(BUILD_DIR)/config/default > $(MANIFEST_DIR)/infrastructure-components.yaml
 	"$(KUSTOMIZE)" build $(BUILD_DIR)/config/supervisor > $(MANIFEST_DIR)/infrastructure-components-supervisor.yaml
 
+## --------------------------------------
+## Verification
+## --------------------------------------
+
+.PHONY: verify
+verify: ## Runs all the verify targets
+	$(MAKE) verify-boilerplate
+	$(MAKE) verify-crds
+	$(MAKE) verify-gen
+	$(MAKE) verify-modules
+	$(MAKE) verify-conversions
+
+.PHONY: verify-boilerplate
+verify-boilerplate: ## Verifies all sources have appropriate boilerplate
+	./hack/verify-boilerplate.sh
+
+.PHONY: verify-crds
+verify-crds: ## Verifies the committed CRDs are up-to-date
```

Comment:
@aartij17 @srm09 @scdubey
Just an observation for a potential follow-up PR. 

I think verify-crds.sh is redundant. The `verify-gen` target is also running `generate-manifests` and then checking if there is a diff.

Reply:
Sure, thanks for pointing it out @sbueringer! 😄 I will create an issue to remove it.